test/sharness/t0060-daemon.sh

```diff
@@ -121,14 +121,30 @@ test_expect_success "daemon with pipe eventually becomes live" '
   test_fsh cat stdin_daemon_out || test_fsh cat stdin_daemon_err || test_fsh cat stdin_poll_apiout || test_fsh cat stdin_poll_apierr
 '
 
-ulimit -n 512
+ulimit -S -n 512
 TEST_ULIMIT_PRESET=1
 test_launch_ipfs_daemon
 
 test_expect_success "daemon raised its fd limit" '
-	grep "ulimit" actual_daemon > /dev/null
+	grep "raised file descriptor limit to 1024." actual_daemon > /dev/null
 '
 
+get_col_four() {
+	awk '{ print $4 }' $1
+}
+
+if [ `uname` == "Linux" ]; then
```

Comment:
The open-fds process would have to start from go-ipfs process, to inherit the fd limit.
Otherwise we are testing nothing.

Other approach would be to make the fd rising separate package, produce a small binary that tries opening files with and without the fd limit raised.